tensorflow_datasets/core/download/kaggle.py

```diff
@@ -91,13 +120,14 @@ class KaggleCompetitionDownloader(object):
 
   def __init__(self, competition_name):
     self._competition_name = competition_name
+    self._kaggle_type = KaggleFile.get_type(self._competition_name)
```

Comment:
As `KaggleFile.get_type` get used outside the class, why not make it a simple function, rather than a class, as suggested previously.